autosarfactory/autosar_ui.py

```diff
@@ -2,22 +2,47 @@
 import tkinter as tk
 import tkinter.ttk as ttk
 import tkinter.font as tkFont
-from ttkthemes import ThemedStyle
+import re
 import os,itertools
-from .autosarfactory import AutosarNode, Referrable
+from ttkthemes import ThemedStyle
+from .autosarfactory import Referrable
+from tkinter import Menu
 
 __resourcesDir__ = os.path.join(os.path.dirname(__file__), 'resources')
 __PAD_X__ = 5 # For some additional padding in the column width
 
 class Application(tk.Frame):
+
+    def selectTheme(self, themeName):
+        style = ThemedStyle(self.__root)
+        if(themeName == 'scidgreen'):
+            self.__current_theme='scidgreen'
+        elif(themeName == 'ubuntu'):
+            self.__current_theme='ubuntu'
+        elif(themeName == 'alt'):
+            self.__current_theme='alt'
+        elif(themeName == 'equilux'):
+            self.__current_theme='equilux'
+        elif(themeName == 'classic'):
+            self.__current_theme='classic'
+        elif(themeName == 'vista'):
+            self.__current_theme='vista'

```

Comment:
Seems like vista has a problem.. I am getting an exception when I select it

Reply:
For me all work. I think is because not all themes are available on al systems/platforms. (For me all work very fast except 'equilux' which is slow in redrawing the widgets).

Reply:
I think, vista is not available in ubuntu.. I have ubuntu 18.04 and I guess vista is just available in windows distribution of ttk.